doc/guides/beyond-GSG.rst

```diff
@@ -303,7 +303,7 @@ Next, run the application.
 
 .. code-block:: console
 
-   west build -t run
+   west build -t run_native
```

Comment:
Again, I don't think native will have any other emulator supported than native itself.

So being able to still just type `west build -t run` makes things much simpler for the ordinary use-case.